tests/vxlan/test_vxlan_ecmp.py

```diff
@@ -450,14 +457,44 @@ def get_t2_ports(duthost, minigraph_data):
         ret_list.append(minigraph_data["minigraph_ptf_indices"][iface])
     return ret_list
 
-def bgp_established(duthost):
+def bgp_established(duthost, ignore_list=[]):
     bgp_facts = duthost.bgp_facts()['ansible_facts']
     for k, v in bgp_facts['bgp_neighbors'].items():
-        if v['state'] != 'established':
-            Logger.info("Neighbor %s not established yet: %s", k, v['state'])
+        if v['state'] != 'established' and k not in ignore_list:
```

Comment:
Incorrect logic here:

1) When we pass ignore list it might look like: `ignore_list = [u'PortChannel105', u'PortChannel102', u'PortChannel1011'....]` - the list of interfaces.
Next, we check `k not in ignore_list`, and `k` is a neighbor's IP adrdess from the next list of keys, e.g. 
`(Pdb) bgp_facts['bgp_neighbors'].keys()`
`[u'10.0.0.61', u'10.0.0.49', u'fc00::22', u'10.0.0.43'....]`
So, `k not in ignore_list` will be always `True`

2) Second, in this list `bgp_facts['bgp_neighbors'].items()` we have both T0 and T2 neighbor's IPs, but we use this function only in TC12 and TC13 where we shutdown/startup only T2 interfaces. We don't need to check T0 for BGP established.